docs/kernel_troubleshooting.rst

```diff
@@ -243,14 +242,22 @@ by issuing the following command:
 
 Make a backup of the file or take a note of the current value of
 ``GRUB_DEFAULT`` somewhere, so you can restore the previous behavior if needed.
-Change the line to ``GRUB_DEFAULT=0``. This will ensure that SecureDrop uses the
-most recent kernel version installed on your server.
+Change the line to ``GRUB_DEFAULT=0``. This configures the bootloader to default
+to loading the most recent kernel version installed on your server.
+
+This change still has to be applied to take effect on the next boot:
+
+.. code:: sh
+
+  sudo update-grub
 
 Safely shut down the *Monitor Server*, remove attached peripherals, and reboot
 it. Verify  that it is working correctly by logging in using your *Admin
 Workstation*. If everything is working as expected, you can make the same change
-to ``/etc/default/grub`` on your *Application Server* as well. You can do so
-from your *Admin Workstation* and reboot the server using the command
-``sudo shutdown -r now``.
+to ``/etc/default/grub`` on your *Application Server* as well. Remember to again
+run the command ``sudo update-grub`` when you are done.
+
+You can make the change on the *Application Server* from your *Admin Workstation*
+and reboot the server using the command ``sudo shutdown -r now``.
```

Comment:
Nit: elsewhere in the guide we use `sudo reboot`, can we use that here, as well, for consistency's sake?